### PR TITLE
Quiet the output on `brew update` for R builds.

### PR DESCRIPTION
**Pull request by craigcitro**
PTAL @BanzaiMan

### CODE REVIEW

File: lib/travis/build/script/r.rb [hunk 1]
```diff
@@ -84,7 +84,9 @@ def setup
             sh.cmd 'sudo chmod 2777 /usr/local/lib/R /usr/local/lib/R/site-library'
 
           when 'osx'
-            sh.cmd 'brew update', retry: true
+            # We want to update, but we don't need the 800+ lines of
+            # output.
+            sh.cmd 'brew update >/dev/null', retry: true
```
Comment (BanzaiMan) [inline lib/travis/build/script/r.rb]: You can also use `echo: false` instead. Either is fine; I'm just pointing it out.

Comment (craigcitro) [inline lib/travis/build/script/r.rb]: oh -- in the generated bash script, it looked like `echo` only controlled whether or not we printed the command before executing it. does it also prevent output from landing in the logs?

Comment (BanzaiMan) [inline lib/travis/build/script/r.rb]: Ah. You are right about that. Carry on.

Comment (craigcitro) [inline lib/travis/build/script/r.rb]: sweet ... then I think this one is ready to go ;)

**Comment by BanzaiMan** [comment]
This, #397, #396, and #395, have been deployed to production.

**Pull request merged**